iree/test/e2e/xla_ops/rng_normal.mlir

```diff
@@ -0,0 +1,11 @@
+func @rng_normal_2d() {
+  %mu = util.unfoldable_constant dense<0.0> : tensor<f32>
+  %sigma = util.unfoldable_constant dense<1.0> : tensor<f32>
+  %shape = util.unfoldable_constant dense<[3, 5]>  : tensor<2xi64>
+  %res = "mhlo.rng_normal"(%mu, %sigma, %shape) : (tensor<f32>, tensor<f32>, tensor<2xi64>) -> tensor<3x5xf32>
```

Comment:
Nit: Numerical checks here aren't the best way to verify samples are form a normal distribution.

Reply:
Thanks Ahmed, this is really a good point. I plot the output and found that it is not normal distribution... :(

I think the reason is that u1 and u2 are not independent. According to Wiki, they should be independent.
> Suppose U1 and U2 are independent samples chosen from the uniform distribution on the unit interval (0, 1).

However, the mhlo.rng_uniform is currently using `seed = 0`, and it results into generating dependent samples. I also tested with the output from mhlo.rng_uniform in C++, and found that it is not a normal distribution.

My current idea is to generate samples at compile time, and it will also address the concern of sin/cos.